pytools/graph.py

```diff
@@ -213,9 +213,17 @@ class CycleError(Exception):
     Raised when a topological ordering cannot be computed due to a cycle.
 
     :attr node: Node in a directed graph that is part of a cycle.
+
+    :attr nodes: List of nodes in a directed graph that are part of a cycle.
```

Comment:
Maybe `path` is a better name. Also state that we claim that `node[i+1]` in `path`  is a successor of `node[i]` (and test that!).

Reply:
Changed it to `paths` and added comment regarding the successors in [47aa316](https://github.com/inducer/pytools/pull/159/commits/47aa3164f3a5e04c9f16afe3bf23fd9cde226275).
Testing this is done in #167 right?